firestore-bundle-server/PREINSTALL.md

```diff
@@ -0,0 +1,31 @@
+Use this extension to build Firestore data bundle files via HTTP requests to a Cloud Function instance, and manage the caching strategy for the built bundle files. Firestore data bundles are static data files built from Firestore documents and query snapshots.
+
+This extension requires your project to be upgraded to the Blaze plan.
```

Comment:
you can remove this line since every extension now requires the Blaze plan

Reply:
Done.